src/pages/auth/login.tsx

```diff
@@ -29,12 +30,25 @@ const LoginPage = () => {
 
     try {
       const response = await UserService.login({ email, password });
-      if (response.message === 'Incorrect email or password') {
+      emailValidation(email, setEmailInvalid);
+      // create e toast message;
+      passwordValidation(password, setPasswordInvalid);
```

Comment:
Change this to something like:

if (!emailValidation(email)) return setEmailInvalid(true);
if (!passwordValidation(password)) return setPasswordInvalid(false);
```

_I personally don't like sending function as attributes. We should call them when we define it (unless some special scenarios)_